README.md

```diff
@@ -1,30 +1,71 @@
-[![CharmHub Badge](https://charmhub.io/github-runner-image-builder/badge.svg)](https://charmhub.io/github-runner-image-builder)
-[![Promote charm](https://github.com/canonical/github-runner-operator/actions/workflows/promote_charm.yaml/badge.svg)](https://github.com/canonical/github-runner-operator/actions/workflows/promote_charm.yaml)
-[![Discourse Status](https://img.shields.io/discourse/status?server=https%3A%2F%2Fdiscourse.charmhub.io&style=flat&label=CharmHub%20Discourse)](https://discourse.charmhub.io)
+# GitHub runner image builder operator
+<!-- Use this space for badges -->
 
-# GitHub runner image builder
+Provide GitHub runner workload embedded snapshot image to the 
+[GitHub runner](https://charmhub.io/github-runner) charm. This charm is deployed as a VM and works
+on top of OpenStack infrastructure.
 
-## Description
+Like any Juju charm, this charm supports one-line deployment, configuration, integration, scaling,
+and more. For Charmed GitHub runner image builder, this includes support for configuring:
+* multi-arch
+* multi Ubuntu bases
+* Juju/Microk8s snap channels
+* external scripts
```

Comment:
Nitpick: I would capitalize the first word of each item in the list, for instance:
```
* Multi-arch
* Multi Ubuntu bases
etc.
```